client/my-sites/plans/jetpack-plans/product-store/items-list/most-popular.tsx

```diff
@@ -2,7 +2,7 @@ import { isJetpackPlanSlug } from '@automattic/calypso-products';
 import clsx from 'clsx';
 import { useStoreItemInfoContext } from '../context/store-item-info-context';
 import { FeaturedItemCard } from '../featured-item-card';
-import { HeroImage } from '../hero-image';
+// import { HeroImage } from '../hero-image';
```

Comment:
I think we are going back to using this component, but in case I am wrong, we should delete this comment 😄